mne/epochs.py

```diff
@@ -3670,7 +3670,7 @@ def average_movements(epochs, head_pos=None, orig_sfreq=None, picks=None,
 
 
 @verbose
-def make_fixed_length_epochs(raw, duration=1., preload=False,
+def make_fixed_length_epochs(raw, id=1, duration=1., preload=False,
```

Comment:
this can break code. Please put the id param just before verbose

Reply:
Ok, so that was my question on the argument order guideline. I've changed it and place the new argument right before verbose.